### DISCUSSION TITLE
Why does my iphone keep showing Living room as leader?

### DISCUSSION THREAD

**Discussion by gbdavidx** [mKeRix/room-assistant#558] [General]
<img width="577" alt="Screen Shot 2021-02-24 at 10 41 30 AM" src="https://user-images.githubusercontent.com/26369204/109049217-dc646c80-768c-11eb-899c-bbc052552c17.png">

the distance is greater then what i have in my configs...
office rasbperry pi
```
global:
  instanceName: office
  integrations:
    - homeAssistant
    - bluetoothClassic
homeAssistant:
  mqttUrl: 'mqtt://192.168.1.123:1883'
  mqttOptions:
    username: user
    password: passwd
bluetoothClassic:
  maxDistance: 10
  addresses:
    - '88:64:40:71:67:6E'
    - '90:81:2A:15:72:11'
    - '90:81:2A:1C:6F:27'
  interval: 6
  scanTimeLimit: 5
  preserveState: false
cluster:
  autoDiscovery: true
  networkInterface: wlan0
  port: 6425
  peerAddresses:
    - 192.168.1.190:6425
    - 192.168.1.191:6425
  weight: 90
```
Living room
```

global:
  instanceName: livingroom
  integrations:
    - homeAssistant
    - bluetoothLowEnergy
homeAssistant:
  mqttUrl: 'mqtt://192.168.1.123:1883'
  mqttOptions:
    username: user
    password: passwd
bluetoothClassic:
  minRssi: -10
  addresses:
    - ’90:81:2A:1C:6F:27’
    - '88:64:40:71:67:6E'
  interval: 6
  scanTimeLimit: 5
  preserveState: false
  maxDistance: 10
cluster:
  autoDiscovery: true
  networkInterface: wlan0
  port: 6425
  peerAddresses:
    - 192.168.1.190:6425
    - 192.168.1.191:6425
  weight: 10
```

**Comment by gbdavidx** [comment]
to Note: my living room pi is at least 20 feet away, by using the rssi values it should be able to tell which one is farther right?

**Comment by mKeRix** [comment]
Yeah, that's how it should work. You can take a look at the [API](https://www.room-assistant.io/guide/api.html) to check the reported values, which should tell you why a state was determined to be like that. I'd guess that it's probably one of these reasons:

- the measured RSSI from `office` is higher
- the last measurement from `office` is outdated
- the connectivity network between `living-room` and `office` isn't great, which could cause measurements to be dropped

You can also compare the states of the two different instances via their respective API endpoints.

Also note that `maxDistance` isn't a valid option for `bluetoothClassic`, that is only for `bluetoothLowEnergy`.
